portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -211,14 +211,29 @@
 
 <!--CUSTOM VIEWS-->
 
+<tr>
+	<td>OPEN_SELECT_FIELD</td>
+	<td>//button[contains(@aria-label, "Open Fields Menu")]</td>
+	<td></td>
+</tr>
 <tr>
 	<td>SELECT_CUSTOM_VIEW</td>
 	<td>//div[contains(@class,'custom-views-selection')]</td>
 	<td></td>
 </tr>
 <tr>
 	<td>SELECT_OPTION_CUSTOM_VIEW</td>
-	<td>//button[contains(@class,'dropdown-item') and contains(text(), '${key_itemName}')]</td>
+	<td>//button[@class='dropdown-item'][normalize-space(text())='${key_itemName}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>INPUT_CUSTOM_VIEW_NAME2</td>
+	<td>//input[contains(@id,'LabelInput')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SAVE_BUTTON</td>
+	<td>//button[@class='dropdown-item'][normalize-space(text())='Save View']</td>
```

Comment:
could you use `SELECT_OPTION_CUSTOM_VIEW` with `key_itemName = 'Save View'`? and delete this path